app/assets/stylesheets/views/_service-toolkit.scss

```diff
@@ -0,0 +1,21 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.service-toolkit {
+  border-bottom: 1px solid $govuk-border-colour;
+  margin-bottom: govuk-spacing(5);
+  padding-bottom: govuk-spacing(3);
+
+  @include govuk-media-query($from: tablet) {
+    margin-bottom: govuk-spacing(9);
+    padding-bottom: govuk-spacing(6);
+  }
+}
+
+.service-toolkit:last-child {
+  border-bottom: none;
+  margin-bottom: 0;
+}
+
+.service-toolkit__description {
+  color: $govuk-secondary-text-colour;
```

Comment:
I wonder if this is necessary? It makes the text a little bit distinct from the other text on the page, but that's already achieved with different text sizes. Personally uncertain, but wondered if you'd considered this.

Reply:
Yeah, I thought it was maybe unnecessary - I'll remove it 👍